rfcs/0044-node-status-and-node-error-backend.md

```diff
@@ -0,0 +1,105 @@
+## Summary
+
+[summary]: #summary
+
+This RFC proposes that we use Google Cloud Logging service to be the backend of the node status/error system. The logs would be stored in the gcloud buckets; and we could use the Log Explorer to search around logs. Metrics can also be created against the logs. For visualization and graphing, we would use Kibana.
+
+## Motivation
+
+[motivation]:#motivation
+
+We need a backend for node staus/error systems. Candidates for the backend include GCloud Logging, AWS stack and Grafana Loki and LogDNA.
+
+## Implementation
+
+[implementation]:#implementation
+
+### Google Cloud Logging
+
+Google Cloud Loggind provides storage and data searching/visualization of logs. It provides a handy command line sdk tool. But unfortunately there's no official support for ocaml binding. The architecture of the Google Cloud Logging is depicted as following: 
+
+![](res/gcloud_logging.png)
+
+For the frontend, each nodes would have 2 command line options to allow them to sed node status/error reports to any specific url: `--node-status-url URL` and `--node-error-url URL`. By default users would send their reports to our backend. They could change the destination by providing their own destination `URL` to the command options.
```

Comment:
Could you reword this slightly:
> By default users would send their reports to our backend.
By default users won't send any status or error reports and can opt-in by providing the `--node-status-url URL` and/or `--node-error-url URL` and our backend would be suggested.

Additionally can we make sure this is configurable from the daemon.json file?

Reply:
Yeah. They are already configurable from the daemon.json file and that's already recorded in the corresponding RFCs.